### ISSUE TITLE
Semantic versioning

### ISSUE THREAD

**Issue by nazar-pc** [oyejorge/less.php#246]

**Comment by oyejorge** [comment]
If it just "looks weird", then I don't see a big reason to change. Here's how to read the less.php versions:

Version numbers can be viewed as LESSJS.LESSPHP

* LESSJS: The first three numbers (1.7.0) represent the less.js version that less.php was built off of
* LESSPHP: Minor changes to make backwards-compatible bug fixes (PATCH)

**Comment by oyejorge** [comment]
Dup of #155

**Comment by nazar-pc** [comment]
Semantic versioning allows you do this, for instance, I'm using following: `MAJOR.MINOR.PATCH+build-BUILD_NUMBER`.
So you can use it like `LESSJS+less.php-LESSPHP` or something similar, just to follow rules.
It is not a big reason, but semantic versioning is really nicely described, well known and self-explaining, so I decided that it might be a good thing to adopt here.
Obviously, it is up to you, as maintainer, to use it or not.

**Comment by oyejorge** [comment]
So instead of ```1.7.0.7```, it sounds like you're suggesting ```1.7.0+build-7````. Is that right?

**Comment by nazar-pc** [comment]
Yes, it might be so (but I'm using it primarily for own projects and build number is always increasing).
Taking third-party thing (as basis or dependency) I usually add shortcut of own project and build number.
So, it might be as `1.7.0+lp7` where `lp` is `less.php`.
Basically you can put anything after `+`, moreover, if you'll try to compare `1.7.0+lp1` with `1.7.0+lp2`, latter will be correctly understood as newer.

**Comment by mwjames** [comment]
> If it just "looks weird", then I don't see a big reason to change. Here's how to read the less.php versions:

Well, those who rely on it as a library mostly understand the interpretation of a `1.7.1` or `1.8.1` as:
```
MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards-compatible manner, and
PATCH version when you make backwards-compatible bug fixes.
```

It is not so much about "looks weird" but to make it easier for users to conceive an awareness about possible breakages or bug fixes based on the common semver.

**Comment by paladox** [comment]
Maybe do something like 

 LESSJS+less.php-LESSPHP 

like was suggestion so like one release for bugfixes and another for new features that is just for less.php and not used by less.js or based on less.js or is based on it.
